en/flight_modes/offboard.md

```diff
@@ -0,0 +1,42 @@
+# Offboard Flight Mode
+
+The vehicle obeys a position, velocity or attitude setpoint provided over MAVLink (often from a companion computer connected via serial cable or wifi). The setpoint can be provided by APIs like [DroneCore](http://dronecore.io/) or [MAVROS](https://github.com/mavlink/mavros).
+
+> **Note** 
+>  * This mode requires position information (GPS or optical flow).
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+>  * The vehicle must be armed before this mode can be engaged.
+
+## Description
+
+Offboard mode is primarily used for controlling vehicle movement and attitude, and supports only a very limited set of MAVLink commands (more may be supported in future). It can be used to:
+* Control vehicle position or velocity (`SET_POSITION_TARGET_LOCAL_NED`). 
+* Control vehicle attitude/orientation (`SET_ATTITUDE_TARGET`)
+
+> **Note** Velocity and attitude commands must be resent regularly at a rate of around (TBD Hz) or PX4 will stop the vehicle. 

```

Comment:
@julianoes Can you confirm the update rate, and which messages it applies to. I would guess velocity and attitude, but not position, on the basis that if a vehicle has been told to go to a location that is OK, but if it is being told to move "forward" you probably want your code to actively command that on a regular basis or stop. (i.e. more defensive approach).

Reply:
The timeout is 0.5 seconds according to:
https://github.com/PX4/Firmware/blob/master/src/modules/commander/commander.cpp#L151

Therefore the update rate needs to be > 2 Hz.

Reply:
Thanks you. I'll fix that!